data/pages.json

```diff
@@ -108,5 +108,10 @@
         "description": "Imposter 404",
         "imageUrl": "https://raw.githubusercontent.com/tsparticles/404-templates/main/__screenshots/imposter.png?raw=true",
         "url": "imposter/404.html"
+    },
+    {
+        "description": "Restriction 404",
+        "imageUrl": "https://raw.githubusercontent.com/RajSpeed/404-templates/restriction/__screenshots/restriction.png?raw=true",
```

Comment:
Please use the right URL, it won't work locally but it will work in production

Reply:
Fixed it, will keep in mind😄